plugins/web/opentelemetry-instrumentation-document-load/src/instrumentation.ts

```diff
@@ -110,6 +110,11 @@ export class DocumentLoadInstrumentation extends InstrumentationBase<unknown> {
           entries
         );
         if (fetchSpan) {
+          fetchSpan.setAttribute(SemanticAttributes.HTTP_URL, location.href);
+          fetchSpan.setAttribute(
+            SemanticAttributes.HTTP_USER_AGENT,
+            navigator.userAgent
+          );
```

Comment:
Is it possible to add a unit-test for that?

Reply:
Thanks for the comment. I tried to add a unit test. But, looks like the unit test in the main branch is broken if run locally before my change. I tried to run "npm run test:browser" and "npm run tdd". Both are failed with the following message. Is it because of my OS or some other settings? Do you have the idea of what went wrong?
![image](https://user-images.githubusercontent.com/21174706/187238712-4cb71a14-4f9e-433f-98d6-daa913c4c05f.png)

I just added the unit test. But, I couldn't run it locally to verify it.

Reply:
@MSNev maybe you know how to make it work?

@tchoi1131  Thanks for your efforts.
Sorry but I have very little experiance with browser instrumentations and tooling :/

Reply:
I haven't sync'd the contrib repo for a while so I have a look later this week.

On the unhandled rejection this can happen when promises (async tests) are used and it throws, so I think the main issue is the first "data" error.

Reply:
It appears that all of the web tests are broken 👎 I've tried several work-arounds as on my machine the "temp" path (which is on a different drive (yes window)) is being included and it fails to compile.

Attempting to roll-back the webpack update from April to see it that "resolves"

Reply:
@MSNev  Thank you so much for following up on this!